src/types/types.ts

```diff
@@ -3,8 +3,14 @@ import { SlackManifest } from "../manifest.ts";
 import { ManifestCustomTypeSchema } from "../types.ts";
 
 export type CustomTypeDefinition =
-  & { callback_id: string }
-  & TypedParameterDefinition;
+  | (
+    & { callback_id: string }
```

Comment:
Is it possible to mark `callback_id` deprecated like we do [here](https://github.com/slackapi/deno-slack-sdk/blob/main/src/functions/types.ts#L131)?